delfin/drivers/hpe/hpe_3par/component_handler.py

```diff
@@ -195,7 +195,7 @@ def list_volumes(self, context):
                     if 'snapCPG' in volume:
                         if orig_pool_id == '':
                             orig_pool_id = volume.get('snapCPG')
-                        else:
+                        elif volume.get('snapCPG') != orig_pool_id:
```

Comment:
componet handler is a mix of rest and ssh handlers.. is it possible to separate rest and ssh client separate.. 
Please consider the comments about need for rest and ssh ? we have to document what is captured from REST and what is from SSH..

Reply:
REST is used to obtain storage system information. The storage system information does not contain status values. Therefore, you can obtain the health status of the storage system only through SSH. The REST and SSH clients in DRIVER are two different client codes and are not mixed.

Reply:
The REST interface is used to obtain storage information, but the storage status is not available. Therefore, the SSH interface must be used to obtain the storage health status.
REST is separate from the SSH client.